docs/next/components/VersionDropdown.tsx

```diff
@@ -6,23 +6,38 @@ import {useVersion} from '../util/useVersion';
 
 import Link from './Link';
 
+function getLibraryVersion(coreVersion) {
+  const [major, minor, patch] = coreVersion.split('.');
+  return parseInt(major) < 1 ? coreVersion : ['0', 16 + parseInt(minor), patch].join('.');
+}
+
+function getLibraryVersionText(coreVersion) {
+  const libraryVersion = coreVersion === 'master' ? 'master' : getLibraryVersion(coreVersion);
+  return libraryVersion === 'master' || coreVersion === libraryVersion
+    ? ''
+    : `/ ${libraryVersion} (libs)`;
```

Comment:
Is it safe to assume users will know what `libs` means?

Reply:
No, but that's what the explanatory text in the dropdown is for.